### PR TITLE
add images and bio to About Me and add icons for links

### PR DESCRIPTION
**Pull request by kierabr1**
I added images and my bio into columns on the About Me page. I changed the text links on the home page to icons.

### CODE REVIEW

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+            <div class="column right">
+                <p class="bio">
+                    Hello, my name is Kiera Brown, and I'm studying computer science at Howard University.
+                    I'm very passionate about devloping tech to help the community. 
+                    Some of my hobbies include sleeping, dancing, cook, and traveling.
+                </p>
+                <p>
+                    Broadening the tech world to underrepresented groups has always been something I’m passionate about. Accessibility and inclusion are extremely important. 
+                    Since I first discovered that I wanted to pursue a career tech, I knew that I would be working in an environment where both women and African Americans were underrepresented, and I wanted to do something to change that.
```
Comment (sbahaud) [inline portfolio/src/main/webapp/about.html:30]: nit: typo - "career in tech" instead of "career tech"

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFunFact() {
+  const funfacts =
```
Comment (sbahaud) [inline portfolio/src/main/webapp/script.js:19]: Looks like the method is not being called from the html. Where you planning to hook it up later?

**Review by sbahaud — APPROVED** [review]
LGTM!

Approving with a few minor comments

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/about.html:30]: Fixed

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (kierabr1) [reply portfolio/src/main/webapp/script.js:19]: Yes, it wasn't working then, but I found a typo, and it works now. I'll send another PR for it.

**Pull request merged**